packages/app/src/DBSearchPage.tsx

```diff
@@ -1712,6 +1728,26 @@ function DBSearchPage() {
                             queryKeyPrefix={QUERY_KEY_PREFIX}
                           />
                           <Group gap="sm" align="center">
+                            {isLive && (
+                              <Tooltip label="Live tail refresh interval">
```

Comment:
What do you think about placing the dropdown on the right-hand side of the datetime picker?